packages/functional-tests/src/app.ts

```diff
@@ -83,4 +85,56 @@ export default class App {
             delete this.activeTests[testName];
         }
     }
+
+
+    private async launchTestBrowser() {
+        while (true) {
+            const tester = MRESDK.Actor.CreateEmpty(this.context, {});
+            var selectedTestName = undefined;

```

Comment:
Prefer `let` over `var`, unless you know you need `var`. tslint should have flagged this.

Reply:
cleaned up